src/Microsoft.DotNet.Darc/Darc/Helpers/UxHelpers.cs

```diff
@@ -123,7 +123,7 @@ public static JObject GetJsonBuildDescription(Build build)
     /// <returns>Description</returns>
     public static string GetTextBuildDescription(Build build)
     {
-        StringBuilder builder = new StringBuilder();
+        var builder = new StringBuilder();
```

Comment:
curious, does VS recommend using this, or the new `StringBuilder builder = new()`?

Reply:
It recommends `var builder = new StringBuilder();`